### PR TITLE
Fix #375: Fix multiple SYNC or DYN CMs of same type

### LINKED ISSUE #375: Cannot add two of the same SYNC/DYN CM to a BEAM accelerator
If you add two of any SYNC CM to the same accelerator, it does not compile. I believe this is due to the way the `USER-SYNCMLCE` type common blocks are included in `COMIN/USER/` and do not depend on the CM name. I believe this also applies to the DYN CMs.

The suggested workaround is to simply duplicate the CM code and give it a new name.

### PR DESCRIPTION
**Pull request by blakewalters**
The user can now compile acclerators containing multiple
instances of any of the SYNC or DYN CMs.  Runs have been
tested successfully for multiple SYNCJAWS and SYNCMLCE.

Also, fixed an annoying output error in DYNVMLC_cm.mortran.

### CODE REVIEW

**Comment by ojalaj** [comment]
Hello @blakewalters, I noticed you have corrected some annoying error in DYNVMLC (and also in corresponding SYNC CMs?). May I ask what kind of error should it solve?

**Comment by blakewalters** [comment]
Hi @ojalaj:  This was an output format statement causing a compilation error for any accelerator with DYNVMLC.  The remainder of these changes just relate to the ability to compile an accelerator having more than one of each type of these CMs.

**Comment by ftessier** [comment]
Just cleaned up to remove the extra merges.

**Comment by ftessier** [comment]
Simply tweaked the commit format and message.

File: HEN_HOUSE/omega/beamnrc/beamnrc_user_macros.mortran [hunk 1]
```diff
@@ -86,8 +86,8 @@ REPLACE {$COVMIN} WITH {10}        "min. no. of particles crossing a scoring"
                                    "average energy/angle only)"
 REPLACE {$MXMED} WITH {12}         "max # of media
 REPLACE {$MXREG} WITH {3000}        "maximum number of regions (absolute)
-REPLACE {$MXSTACK} WITH {10000}    "maximum stack (much less if no brem split)
-REPLACE {$MAXBRSPLIT} WITH {2000}  "maximum number of bremsstrahlung splits
+REPLACE {$MXSTACK} WITH {1000000}    "maximum stack (much less if no brem split)
+REPLACE {$MAXBRSPLIT} WITH {500000}  "maximum number of bremsstrahlung splits
```
Comment (ftessier) [inline HEN_HOUSE/omega/beamnrc/beamnrc_user_macros.mortran]: @blakewalters can you confirm that committing these increases in default sizes is intentional?

Comment (blakewalters) [reply HEN_HOUSE/omega/beamnrc/beamnrc_user_macros.mortran]: Absolutely unintentional @ftessier; these are the settings I routinely use.  Thanks for catching this!  I'll return them to their original defaults.

Comment (ftessier) [reply HEN_HOUSE/omega/beamnrc/beamnrc_user_macros.mortran]: All right, just push the change as a new commit on top, as discussed!

**Comment by ftessier** [comment]
Ironed out the merge bubble.

**Pull request merged**